pytrip/tripexecuter/projectile.py

```diff
@@ -30,55 +30,61 @@ class Projectile(object):
     """
     Object holding projectile specific data.
     """
-
     # list of projectile name - charge and most common isotope.
-    _projectile_defaults = {"H": (1, 1),
-                            "He": (2, 4),
-                            "Li": (3, 7),
-                            "C": (6, 12),
-                            "O": (8, 16),
-                            "Ne": (10, 20),
-                            "Ar": (16, 40)}
-
-    z = -1
-    a = -1
-    iupac = ""  # chemical symbol (parsed by TRiP), "H", "He", "C", "O", "Ne" ....
-    name = ""   # cleartext name "Protons", "Carbon-12 ions", "Antiprotons", ...
-
-    def __init__(self, symb, name="", z=0, a=0):
+    projectile_defaults = {
+        "H": (1, 1),
+        "He": (2, 4),
+        "Li": (3, 7),
+        "C": (6, 12),
+        "O": (8, 16),
+        "Ne": (10, 20),
+        "Ar": (16, 40)
+    }
+
+    def __init__(self, iupac_symbol: str, a: int = 0, z: int = 0, name: str = ""):
         """
-        :param symb: IUPAC symbol "H", "He", "Li", "C" ...
+        :param iupac_symbol: IUPAC symbol "H", "He", "Li", "C" ..., from projectile_defaults list
```

Comment:
", from projectile_defaults list" is obsolete. Ill delete this.
Firstly I've thought that iupac_symbol must be one from projectile_defaults, then I looked at:
http://bio.gsi.de/DOCS/TRiP98/PRO/DOCS/trip98dataobjects.html#Projectiles